packages/i/ispc/package.yml

```diff
@@ -1,26 +1,48 @@
 name       : ispc
 version    : 1.19.0
-release    : 9
+release    : 10
 source     :
     - git|https://github.com/ispc/ispc.git : v1.19.0
+homepage   : https://github.com/ispc/ispc
 license    : BSD-3-Clause
 component  : programming
 summary    : Intel SPMD Program Compiler
 description: |
     ispc is a compiler for a variant of the C programming language, with extensions for single program, multiple data programming. Under the SPMD model, the programmer writes a program that generally appears to be a regular serial program, though the execution model is actually that a number of program instances execute in parallel on the hardware.
-clang      : yes
 builddeps  :
-    - pkgconfig(LLVMSPIRVLib)
     - pkgconfig(libze_loader)
+    - pkgconfig(tbb)
     - vc-intrinsics
+    - spirv-llvm-translator-15-devel
+    - llvm-clang-15-devel
+rundeps    :
+    # For libomp.so
+    # The difference between LLVM 15 and 16 libomp.so is not large enough to
+    # prevent us from using it, and the symbols we use have basically been there
+    # forever.
+    - llvm
+environment: |
+    export CMAKE_PREFIX_PATH=/usr/lib64/llvm15
+    export CXXFLAGS="$CXXFLAGS -lzstd -lz"
+
+    if [ -f /usr/bin/ccache ]; then
+        export CMAKE_C_COMPILER_LAUNCHER=ccache
+        export CMAKE_CXX_COMPILER_LAUNCHER=ccache
+    fi
 setup      : |
+    %patch -p1 -i $pkgfiles/0001-Fix-build-with-GCC-13-and-possibly-Clang-15
+
     %cmake_ninja \
       -DARM_ENABLED=OFF \
-      -DISPC_NO_DUMPS=ON \
       -DISPC_INCLUDE_EXAMPLES=OFF \
+      -DISPC_INCLUDE_TESTS=OFF \
+      -DISPC_INCLUDE_XE_EXAMPLES=OFF \
+      -DISPC_NO_DUMPS=ON \
       -DISPC_OPAQUE_PTR_MODE=OFF \
+      -DISPC_PREPARE_PACKAGE=OFF \
+      -DISPCRT_BUILD_TESTS=OFF \
       -DXE_ENABLED=ON \
-      -DXE_DEPS_DIR=/usr
+      -L
```

Comment:
`-L`? What does this do

Reply:
It's okay if the answer is "magically makes it build", I'm just curious

Reply:
Oh no, it doesn't make stuff build.  `-L` outputs all the CMake variables that are set. It just makes it easier to debug in the future because the CMake variables can change drastically without notice in these projects.